content/docs/introducing-jsx.md

```diff
@@ -6,17 +6,17 @@ prev: hello-world.html
 next: rendering-elements.html
 ---
 
-Consider this variable declaration:
+Considera la declaración de esta variable:
 
 ```js
 const element = <h1>Hello, world!</h1>;
 ```
 
-This funny tag syntax is neither a string nor HTML.
+Esta divertida sintaxis de etiquetas no es ni un string ni HTML.
```

Comment:
What do you think about `curiosa` instead of `divertida`? I feel like it fits the context better since this is an introduction, it is to incentive the reader curiosity.

Reply:
What about translate string as "cadena de texto"? or use *string*, to remark the word written in other language.

Reply:
I think a string is a string, it's taught as a concept with that name and it's easily understood by any programmer independently of the language.

Reply:
I agree with you @gariasf, technical dev terms don't need translation.